app/docs/0.11.x/getting-started/quickstart.md

```diff
@@ -20,7 +20,8 @@ supports PostgreSQL and Cassandra).
     Issue the following command to [start][CLI] Kong:
 
     ```bash
```

Comment:
I thought about that, but especially for a "quick start" I really focussed on keeping the page as short as possible, such that page on a first glance keeps the impression of very short and minimal steps.